rixs/tests/test_process2d.py

```diff
@@ -0,0 +1,35 @@
+from rixs import process2d
+import numpy as np
+from numpy.random import rand
+from numpy.testing import assert_array_almost_equal
+
+
+def test_curvature_fit():
```

Comment:
These tests are great, however I would like to see all functions have at least one test.

Reply:
Agreed, but I wouldn't necessarily block merging on this in _this_ case. (Internal DAMA libs are held to a different standard in that respect.)